ci/test.sh

```diff
@@ -61,6 +61,9 @@ echoinfo "Test complete"
 
 echoinfo "Uploading test results"
 
+# We don't care too much if this intermittently fails,
+# so don't fail the entirety of CI if anything below this line fails using set +e
+set +e
```

Comment:
Oh man, why didn't we have this before :smile:

Reply:
you can remove this from here because of https://github.com/stargate/stargate/pull/1832, right?

Reply:
This is `v2` branch, not sure if it reached the `v1` state as well. But sure we can remove so we don't have this in this commit.

Reply:
The PR I listed above is on `master` which means it should get merged down as part of regular release process.

Reply:
Yes I can remove this - I just wanted it to be here immediately without having to do all the merging of master into v2, etc